tests/pubsub/test_pubsub.py

```diff
@@ -1,10 +1,10 @@
-from hathor.pubsub import HathorEvents, PubSubManager
+from hathor.pubsub import EventArguments, HathorEvents, PubSubManager
 from tests.unittest import TestCase
 
 
 class PubSubTestCase(TestCase):
-    def test_duplicate_subscribe(self):
-        def noop():
+    def test_duplicate_subscribe(self) -> None:
+        def noop(event: HathorEvents, args: EventArguments) -> None:
```

Comment:
Have you added arguments in the `noop()` function?

Reply:
Yes, the callable used in `PubSub.subscribe` is fully typed and expects those arguments. This was not enforced before because this test was not typed.

Reply:
Got it. Thanks!